Bugsnag/KSCrash/Source/KSCrash/Recording/BSG_KSSystemInfo.m

```diff
@@ -70,7 +70,7 @@ static inline bool is_jailbroken() {
  *
  * https://opensource.apple.com/source/xnu/xnu-7195.81.3/libsyscall/wrappers/system-version-compat.c.auto.html
  */
-static NSDictionary * bsg_systemversion() {
+__attribute__((unused)) static NSDictionary * bsg_systemversion() {
```

Comment:
Marking this function as unused is misleading.

Would be clearer to surround in `#if !TARGET_OS_SIMULATOR`

Reply:
good point, updated!

Reply:
Thanks @keith 😃